cfme/middleware/__init__.py

```diff
@@ -1,8 +1,39 @@
-from cfme.web_ui import CheckboxTable, toolbar as tb
 from functools import partial
 
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import CheckboxTable, toolbar as tb
+from utils.browser import ensure_browser_open
+from cfme.common import Validatable, SummaryMixin
+from utils.log import logger
```

Comment:
Can you please reorder the imports?

```python
import a_some_standard_library
import b_some_standard_library
from a_standard_library import a, b
from another_standard_library import c, d

from cfme.common import ...
from cfme.fixtures import ...
...
```

This was not enforced much but you find the usefulness of the import ordering once you get to modify them or rebase :)